### DISCUSSION TITLE
Anyway to discover FinderID now?

### DISCUSSION THREAD

**Discussion by dryMAILman** [GerryDazoo/Slinger#316] [General]
### Discussed in https://github.com/GerryDazoo/Slinger/discussions/312

<div type='discussions-op-text'>

<sup>Originally posted by **tryin100** November 17, 2022</sup>
Now that Slingbox has been shut down is there anyway to discover the Finder ID of the slinbox 500s that I have - or, are they now just landfill?
thanks</div>

**Comment by dryMAILman** [comment]
SlingPlayer for Fire TV worked today (11/17/2022).  Out of home.  Not on a LAN!  The on screen remote also worked…for a time.  Now I’m getting the dreaded  “Unable to locate your Slingbox.”  error.  Did I brick my 500?  Could the Fire TV app be a helpful window or guide to this project?

**Reply by GerryDazoo** [reply]
Run the slingbox_server  without setting the IP address for the slingbox and it will probe your LAN and print out the IP, port and FinderID. You do not need the finderid to make my server work.
